staging/src/k8s.io/apiextensions-apiserver/hack/openapi-violation.list

```diff
@@ -0,0 +1,7 @@
+API rule violation: names_match,k8s.io/apimachinery/pkg/apis/meta/v1,APIResourceList,APIResources
```

Comment:
@marun is this commit still necessary? We should have the same openapi validation as upstream.

Reply:
Dropped